plugins/org.eclipse.n4js/src/org/eclipse/n4js/utils/N4JSLanguageUtils.xtend

```diff
@@ -844,6 +845,11 @@ public class N4JSLanguageUtils {
 			&& value.compareTo(N4JSGlobals.INT32_MAX_VALUE_BD) <= 0;
 	}
 
+	/** Checks presence of {@link AnnotationDefinition#POLYFILL} annotation. See also {@link N4JSLanguageUtils#isStaticPolyfill(AnnotableElement) }*/
+	def static boolean isNamespaceElement(AnnotableElement astElement) {
+		return AnnotationDefinition.POLYFILL.hasAnnotation( astElement );
```

Comment:
Name of this method does not seem to match its behavior. (probably a copy/paste error?)